### ISSUE TITLE
'Strong name validation failed' for NuGet.Packaging.Core v3.4.3

### ISSUE THREAD

**Issue by drxs3v3n** [aelij/RoslynPad#87]
I clone the RoslynPad repo, build solution in visual studio 2017, it restores all the nuget packages, I run the compiled exe, I pick a nuget package to pull in using RoslynPad, and then I get an exception: "'Strong name validation failed' for NuGet.Packaging.Core v3.4.3".

I've found these two links where the strong name validation for NuGet.Packaging.Core is a problem for some people.  I've tried 3 computers so far using vs2017 and I can't figure out how to pull the right NuGet.PackageManagement that does not have the strong name validation probllem.  I know someone has to know how to fix this, so I'm posting here in hopes that someone has built RoslynPad and figured it out.

Here are the two links I've seen this discussed on:
https://github.com/NuGet/Home/issues/2690
https://github.com/NuGet/Home/issues/3596

Part of the reason I can't figure out how to fix this, is I don't see where the dlls downloaded by nuget are stored at.  I'm used to seeing a packages folder, and I've researched how you move it but don't see those settings anywhere.  I've also looked around at this place: %localappdata%\NuGet\v3-cache\, but can't find dlls there.

I've deleted out the v3-cache, and tried several things to force it to pull new packages from Nuget.

**Comment by aelij** [comment]
Try looking for the package cache under %userprofile%\.nuget
I'll be upgrading to NuGet 4 soon

**Comment by drxs3v3n** [comment]
I used your comment to get it working.  I went to %userprofile%\.nuget\packages\nuget.packaging.core\3.4.3\lib\net45, and replaced all the files there with files downloaded from here: https://www.nuget.org/packages/NuGet.Packaging.Core/3.4.3.  Then rebuild and I'm able to add nuget packages.  I also ran sn -v on the NuGet.Packaging.Core.dll and gives the workable result of "Assembly 'NuGet.Packaging.Core.dll' is valid".

I know this is nuget's problem, but thanks for the asistence.
